tools/walletextension/README.md

```diff
@@ -27,19 +27,25 @@ unencrypted. If the data is not particularly sensitive, it can also be run in an
 
 ## Usage
 
-1. Clone this repository: https://github.com/obscuronet/go-obscuro
+1. Precompiled binaries for the wallet extension are available for macOS AMD64 and Windows AMD64 on the project's 
+   [releases page](https://github.com/obscuronet/go-obscuro/releases/)
 
-2. Build the wallet extension binary by running the following from the root of the repository:
-  ```
-  cd tools/walletextension/main
-  go build -o wallet_extension .
-  ```
-  This will build the wallet extension binary under `tools/walletextension/main/wallet_extension`
+   * For other OSes and architectures, the wallet extension binary can be built by cloning the 
```

Comment:
Tempted to make it 'see below for how to build for other architectures' or something and bury it at the bottom, just to get anything scary looking off the main path?

Reply:
Fixed